py4cast/datasets/base.py

```diff
@@ -808,6 +807,86 @@ def projection(self):
         return func(**self.projection_kwargs)
 
 
+@dataclass(slots=True)
+class Settings:
```

Comment:
It is nice to see a generic Settings ! Maybe just add doctring to describe the class.

Reply:
ok